modules/c3-assisted-installer-overview.adoc

```diff
@@ -32,15 +32,23 @@ The installation process subsequently provisions three control plane (master) no
 
 .Main steps
 
-The main steps of the procedure are as follows:
+The following workflow describes a high-level outline for installing an {product-title} on {oci-c3-short} using the {ai-full}:
 
-. Preparing the {oci-c3-short} bastion server.
+. Prepare the {oci-c3-short} bastion server (Oracle).
 
-. Running the Terraform script via the Home region. 
+. Run the Terraform script via the Home region (Oracle).
 
-. Preparing the {product-title} image for {oci-first-no-rt}. 
+. Generate the {product-title} image in the {ai-full} web console (Red Hat). 
```

Comment:
Per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces), there should be a non breaking space in between Red and Hat:
```suggestion
. Generate the {product-title} image in the {ai-full} web console (Red{nbsp}Hat). 
```